src/Cake.Core/IO/ProcessRunner.cs

```diff
@@ -17,16 +17,19 @@ public sealed class ProcessRunner : IProcessRunner
     {
         private readonly ICakeEnvironment _environment;
         private readonly ICakeLog _log;
+        private readonly IFileSystem _fileSystem;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ProcessRunner" /> class.
         /// </summary>
         /// <param name="environment">The environment.</param>
         /// <param name="log">The log.</param>
-        public ProcessRunner(ICakeEnvironment environment, ICakeLog log)
+        /// <param name="fileSystem">The file system.</param>
+        public ProcessRunner(ICakeEnvironment environment, ICakeLog log, IFileSystem fileSystem)
```

Comment:
Can we move the fileSystem argument first here? We put it first everywhere else and this makes my OCD tingle 😄

Reply:
My OCD says put new last, but your OCD is stronger than mine, so I'll address ;)